prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/SC/sc-phd.yml

```diff
@@ -0,0 +1,91 @@
+---
+- name: "sc-phd"
+  description: "South Carolina Public Health Department"
+  jurisdiction: "STATE"
+  stateCode: "SC"
+  countyName: null
+  filters: []
+  featureFlags: null
+  keys: null
+  senders: []
+  receivers:
+    - name: "full-elr"
+      organizationName: "sc-phd"
+      topic: "full-elr"
+      customerStatus: "active"
+      translation: !<HL7>
+        schemaName: "classpath:/metadata/hl7_mapping/receivers/STLTs/SC/SC-receiver-transform"
```

Comment:
@oslynn Hey FYI, you'll also need `.yml` at the end when you actually push this setting.

Reply:
```suggestion
        schemaName: "classpath:/metadata/hl7_mapping/receivers/STLTs/SC/SC-receiver-transform.yml"
```